usdoj-crt/beta-ada
# Adding minimization optimization to webpack

Currently, our live JS files are not minimized. I'm not sure exactly why not, as the Webpack mode should be set to production when building. In the meantime, I added a flag to enable minimization of JS assets using the built in Terser plugin. 

To Test:

1. Pull down the branch locally
2. Run npm run test:ui
3. All the tests should pass
4. View the code source on the Federlalist link. Check the /assets/dist/accordion-compiled file to make sure that it has been compressed.
